client/src/scss/_variables.scss

```
@@ -16,3 +16,5 @@ $accent-1-bg: #b03045;
 $accent-1-text: #fff;
 $accent-2-bg: #4e459d;
 $accent-2-text: #fff;
+
+$blurple: #7289DA;
```

```suggestion
$blurple: #7289da;
```
Lower case for consitency with the rest of the file.